index.html

```diff
@@ -1265,6 +1282,9 @@ <h3><dfn>close()</dfn> method</h3>
 
     <ol>
       <li>Let |promise| be [=a new promise=].
+      <li>
+        If [=this=].{{SerialPort/[[state]]}} is not `"opened"`, reject |promise| with an
+        "{{InvalidStateError}}" {{DOMException}} and return |promise|.
```

Comment:
Good catch. This is what the implementation did but I never specified that behavior.